src/styles/components/_sandpack.scss

```diff
@@ -0,0 +1,163 @@
+// the article container
+.sandpack {
+  header {
+    color: var(--foreground-color);
+
+    h2 {
+      position: relative;
+      font-size: 1.5625rem; // 25px/16
+      font-family: Viga, sans-serif;
+      line-height: 2.125rem; // 34px/16
```

Comment:
`line-height` should be unitless 🙃 
I think the `font-size` changes from other PRs will apply here, as well. And do we have a font-family map? I don't remember if we set one up.

Reply:
Line-height: ✅

We have a font size map, but 25px is not part of it (that was something I added in the font audit PR). We never set up a font-family map. 🤔

Reply:
https://github.com/sparkbox/accessible-components/pull/105/commits/1fa508dd12b7bc45b33ba214d1f04fd1ea5dae89

Reply:
Okay, just make a note of the font-size, then. Whichever PR gets merged first, the other one will need to update it to match the end result we want.